docs/deploy/onprem.md

```diff
@@ -164,16 +165,30 @@ To install lakeFS with Helm:
 
    To configure a load balancer to direct requests to the lakeFS servers you can use the `LoadBalancer` Service type or a Kubernetes Ingress.
    By default, lakeFS operates on port 8000 and exposes a `/_health` endpoint that you can use for health checks.
-   
+
    💡 The NGINX Ingress Controller by default limits the client body size to 1 MiB.
    Some clients use bigger chunks to upload objects - for example, multipart upload to lakeFS using the [S3-compatible Gateway](../understand/architecture.md#s3-gateway) or 
    a simple PUT request using the [OpenAPI Server](../understand/architecture.md#openapi-server).
    Checkout Nginx [documentation](https://kubernetes.github.io/ingress-nginx/user-guide/nginx-configuration/annotations/#custom-max-body-size) for increasing the limit, or an example of Nginx configuration with [MinIO](https://docs.min.io/docs/setup-nginx-proxy-with-minio.html).
    {: .note }
-   
+
 </div>
 </div>
 
+## Secure connection
+
+It is recommended that TLS/SSL termination by your infrastructure's load balancer or cluster manager.
+
+To configure your local lakeFS to listen and serve with HTTPS, update `config.yaml` with the additional `tls` section:
```

Comment:
```suggestion
To configure your local lakeFS to listen and serve with HTTPS, update the lakeFS config yaml with the additional `tls` section:
```

Reply:
we don't explain on each page how we translate the config.yaml entries from/to env vars names. In this page the guide was based on config.yaml so I've explained how to update the file to include the relevant settings.